### PR TITLE
AR & AS: Create app skeleton with node/express/p5/eslint

### PR DESCRIPTION
**Pull request by sa-mcquanzie**

### CODE REVIEW

File: bin/www [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * Event listener for HTTP server "listening" event.
+ */
+
+function onListening() {
+  var addr = server.address();
+  var bind = typeof addr === 'string'
+    ? 'pipe ' + addr
+    : 'port ' + addr.port;
+  debug('Listening on ' + bind);
+}
```
Comment (danroche10) [inline bin/www:90]: Could someone explain this file as I have seen most of it before ?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+app.set('view engine', 'ejs');
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.use('/', indexRouter);
+app.use('/users', usersRouter);
```
Comment (danroche10) [inline app.js:22]: What will the /users route be for?

**Review by danroche10 — COMMENTED** [review]
Hey guys,

This looks ok for backend but I think most of the backend stuff won't be needed for MVP.

I think backend stuff could be done in a few lines and then expanded upon.

If you think it's all necessary then it would be good if you could take the group through it so we're all on same page. Atm it's not clear what a lot of this is doing.

For MVP, I think it's best to focus on setting up the main class files, the unit tests and the feature tests.

Let me know what you think

File: app.js (hunk 2 above)
Comment (sa-mcquanzie) [reply app.js:22]: If we add a user model later as discussed then all the actions associated with our User model will live under this route

**Pull request merged**